db-migrations/29_protocol.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE version_endpoints ADD COLUMN protocol VARCHAR(32);
+ALTER TABLE model_endpoints ADD COLUMN protocol VARCHAR(32);
```

Comment:
why do we need protocol field in model_endpoints?

Reply:
It's necessary to allow consumer of model endpoint to know the protocol supported without having to know its underlying version endpoint. Having the value in DB is just for convenience.